app/src/main/res/layout/dashboard_categories_concat.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
You aren't using any actual databinding functionality in this layout, why not use ViewBinding instead of DataBinding for it?

Reply:
hmm, the `SimpleDataBindingAdapter` isn't setup to be able to leverage ViewBinding at the moment.

Reply:
hold off on getting rid of data binding for the time being, the base adapters probably need to be updated for it first.

Reply:
this doesn't need to be a databinding layout